conda_smithy/tests/test_lint_recipe.py

```diff
@@ -107,7 +107,7 @@ def test_test_section_with_recipe(self):
             lints = linter.lintify({}, recipe_dir)
             self.assertIn(expected_message, lints)
 
-            with open(os.path.join(recipe_dir, 'run_test.py'), 'w') as fh:
+            with io.open(os.path.join(recipe_dir, 'run_test.py'), 'w') as fh:
```

Comment:
On Python 3, they are identical. On Python 2, `io.open` is basically a backport of Python 3's `open`.